internal/providercache/installer.go

```diff
@@ -292,6 +294,41 @@ NeedProvider:
 		}
 	}
 
+	// We'll remember our selections in a lock file inside the target directory,
+	// so callers can recover those exact selections later by calling either
```

Comment:
Is there an "or" to this "either" that got missed?

Reply:
oh heh yes... I removed the thing that was in the "or" here because it turned out to be superfluous, but my comment editing was apparently too hasty.